src/components/primitives/Heading.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Text } from '../';
+
+export default React.forwardRef((props, ref) => {
+	return <Text variant="heading" as="h1" ref={ref} {...props} />;
```

Comment:
```suggestion
	return <Text as="h1" {...props} variant="heading" ref={ref} />;
```

Otherwise, it's possible to overwrite variant.